docs/vmanomaly.md

```diff
@@ -135,3 +135,59 @@ It is also possible to split up config into multiple files, just list them all i
 ```sh
 python3 -m vmanomaly model_prophet.yaml io_csv.yaml scheduler_oneoff.yaml
 ```
+
+### Licensing
+
+Starting from v1.5.0 vmanomaly requires a license key to run. You can obtain a trial license
+key [here](https://victoriametrics.com/products/enterprise/trial/).
+
+The license key can be passed via the following command-line flags:
+```
+  --license LICENSE     See https://victoriametrics.com/products/enterprise/
+                        for trial license
+  --license-file LICENSE_FILE
+                        See https://victoriametrics.com/products/enterprise/
+                        for trial license
+  --license-verify-offline LICENSE_VERIFY_OFFLINE
```

Comment:
NIT: don't follow the need of this flag

Reply:
This flag is used to force offline verification instead of default online verification.

Reply:
i guess user will have the same question :) need to describe it

Reply:
Updated description